.github/scripts/deny-git-deps.py

```diff
@@ -34,15 +35,16 @@ def check_dep(dep, used_by):
 # Check the workspace dependencies that can be inherited:
 for dep in workspace.dependencies:
 	check_dep(dep, "workspace")
-
 	if workspace.crates.find_by_name(dep.name):
 		if dep.location != DependencyLocation.PATH:
 			errors.append(f'🚫 Workspace must use path to link local dependency {dep.name}')
 
 # And the dependencies of each crate:
 for crate in workspace.crates:
 	for dep in crate.dependencies:
-		check_dep(dep, crate.name)
+		# Allow git dependencies in `[dev-dependencies]` sections.
+		if dep.kind != "dev":
+			check_dep(dep, crate.name)
```

Comment:
IMO we should publish `txtt` before merging.

Reply:
I published a tool here:
https://crates.io/crates/substrate-txtesttool

Reply:
Nice, thank you! Will get back with the changes.